Documentation/ApiOverview/Environment/ApplicationContext.rst

```diff
@@ -0,0 +1,383 @@
+:orphan:
+
+..  include:: /Includes.rst.txt
+..  index:: ! Application Context
+
+..  _application-context:
+
+===================
+Application Context
+===================
+
+In previous TYPO3 versions, error messages were displayed unfiltered in the
+frontend and often revealed security-relevant data such as database access,
+e-mail addresses, SMTP access and similar to the public. The so-called
+`ApplicationContext` was introduced to prevent this. These contexts can
+pre-configure TYPO3 to a certain extent and can also be queried again at
+various points such as the site configuration, e.g. to provoke a different
+:ref:`base variants <sitehandling-baseVariants>`.
+
+TYPO3 is delivered with 3 different modes, which affect the behaviour of TYPO3
+as follows:
+
+*   Production
+
+    *   Only errors are logged.
+    *   There is no logging of obsolete (deprecated) function calls.
+    *   In the event of an error, the frontend only displays:
+        `Oops, an error occurred! Code: {code}`.
+    *   The :ref:`Dependency injection <Dependency-Injection>` cache can only
+        be emptied using the Flush Cache button in the install tool.
+    *   Calling up the install tool menu items requires the additional entry
+        of a password.
```

Comment:
```suggestion
        of a password ("sudo mode").
```

Adding the name for it, so people can look for it if needed.

Reply:
Just added